tasks.c

```diff
@@ -4730,6 +4943,10 @@ BaseType_t xTaskIncrementTick( void )
 
     traceENTER_xTaskIncrementTick();
 
+    #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) )
+        UBaseType_t uxSavedInterruptStatus = kernelENTER_CRITICAL_FROM_ISR();
+    #endif /* #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) ) */
+
```

Comment:
```suggestion
    #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) )
        UBaseType_t uxSavedInterruptStatus;
    #endif /* #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) ) */

    traceENTER_xTaskIncrementTick();

    #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) )
        uxSavedInterruptStatus = kernelENTER_CRITICAL_FROM_ISR();
    #endif /* #if ( ( portUSING_GRANULAR_LOCKS == 1 ) && ( configNUMBER_OF_CORES > 1 ) ) */
```

Suggestion for ISO C90 requirement that all variable declarations must be at the beginning of a block.